src/Nova/Note.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Notes\Nova;
+
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\BelongsTo;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\ID;
+use Laravel\Nova\Fields\Markdown;
+use Laravel\Nova\Fields\MorphTo;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Http\Requests\NovaRequest;
+use Laravel\Nova\Panel;
+use Tipoff\Support\Nova\Resource;
+
+class Note extends Resource
+{
+    public static $model = \Tipoff\Notes\Models\Note::class;
+
+    public static $title = 'content';
+
+    public static $search = [
+        'id',
+    ];
+
+    public static $group = 'Media';
+
+    public function fieldsForIndex(NovaRequest $request)
+    {
+        return [
+            ID::make()->sortable(),
+            Text::make('Content'),
+            MorphTo::make('Noteable')->sortable(),
+        ];
+    }
+
+    public function fields(Request $request)
+    {
+        return [
+            Markdown::make('Content')->required(),
+
+            MorphTo::make('Noteable')->types([
+                User::class,
+                Contact::class,
+                Customer::class,
+                Order::class,
+                Booking::class,
+                Game::class,
+                Block::class,
+                Slot::class,
+            ]),
```

Comment:
@sl0wik What is the best way to handle this?

Reply:
@devjk1 added the nova helper in now and is re-running the tests